CorsixTH/Lua/languages/simplified_chinese.lua

```diff
@@ -2098,7 +2098,7 @@ vip_names = {
   "来自荷兰的伯那德国王",
   "缅甸民主党领袖：昂山苏蒂",
   "克朗伯先生",
-  "比利先生",
+  "兰斯洛特·斯普拉特爵士",
```

Comment:
Replace this block as below:

```lua
vip_names = {
  health_minister = "卫生部部长",
  [1] = "伟大的查普顿市长", -- the rest is better organized in an array.
  [2] = "南丁格尔",
  [3] = "来自荷兰的伯那德国王",
  [4] = "缅甸民主党领袖：昂山苏蒂",
  [5] = "克朗伯先生",
  -- [6] requires replacement (English uses 'Sir Lancelot Spratt')
  [7] = "克劳福议员",
  [8] = "罗尼",
  [9] = "一个超级联赛球星",
  [10] = "拉里普罗斯特",
}
```

Reply:
This does mean there'll be a 10% chance of an untranslated string appearing in the fax machine message when playing in Simplified/Traditional Chinese, but perhaps it's better to show an untranslated string than a badly translated string.

Reply:
Lokpro will likely pick up on it soon enough if they're still watching for translations; shouldn't be a massive issue.